shipment_advice/models/shipment_advice.py

```diff
@@ -444,18 +444,25 @@ def _unplan_undone_moves(self):
         ).filtered(lambda m: m.state not in ("cancel", "done") and not m.picked)
         moves_to_unplan.shipment_advice_id = False
 
-    def _postprocess_action_done(self):
+    def _postprocess_action_done(self, backorder_policy):
```

Comment:
I would make the new argument a keyword argument w/ a default value for backward compatibility, and avoid breaking inheriting modules that already use this function somewhere else